SDWebImage/SDWebImageWebPCoder.m

```diff
@@ -230,10 +230,15 @@ - (UIImage *)incrementallyDecodedImageWithData:(NSData *)data finished:(BOOL)fin
 - (UIImage *)decompressedImageWithImage:(UIImage *)image
                                    data:(NSData *__autoreleasing  _Nullable *)data
                                 options:(nullable NSDictionary<NSString*, NSObject*>*)optionsDict {
-    // Decompress can help pre-draw the image and transfer the backing store to render process.
-    // Well, it can reduce the `App process memory usage` from Xcode, because the backing store is in `Other process` (render process). But it does not help for total memory usage for device.
```

Comment:
@dreampiggy I need to confirm, what's the time of `reduce the `App process memory usage` from Xcode` you mean? I think you mean the time is when image will show on screen? Actually, we do two decompress, one is `WebP` data to `RGB`, another is to decompress by call `decompressedImageWithImage`.

Reply:
...Maybe you don't know why SDWebImage have force decode. Check that blog again and you'll understand.

The decode logic, is happend to decompressed the compressed data buffer (for example, JPEG), to the bitmap buffer (RGBA pixel array).

However, Image/IO, the system decoder, use a lazy policy, when you call `CGImageSourceCreateImageAtIndex()`, it only parse the bistream, without generate the bitmap array buffer and allocate the memory.

Only when you call `CGDataProviderRetainBytePtr` (or the public API `CGDataProviderCopyData`, they start to generate and allocate the bitmapbuffer. However, this step, by default the pipeline, will happened during the Core Animation render logic (by `CA::render::commit`), and it trigger the decoding in the main queue. Which may reduce the frame rate.

To solve that issue, we and many many Image loading framework (including YY, Kingfisher, etc), use a pre-trigger of decoding, which force to decode that in advance and increate the frame rate. However, it's a trade-off between CPU && RAM. So this is a option to let user disable this feature totally.

Reply:
But things does not apply the same for WebP images. For WebP, we call libwebp's `WebDecode()` function, which immediatelly decode and allocate the bitmap array buffer. It's not lazy at all. So, even if this image was rendered and called by Core Animation to get the bitmap buffer, it don't need to block main queue. So no extra performance  cost.

But things is interesting, because of the way we trigger force decode. (To trigger that `CGDataProviderRetainBytePtr `, there are many ways. Draw on bitmap canvas will implicitly call that method). When you call `CGContextDrawImage`, it have another side-effect. It will call `vm_copy`, to copy your bitmap buffer backing store, from the App Process, into the Render Process. 

So you will see after call the force decode, you App memory (showing on Xcode's Memory chart) become lower. But the total memory does not change. It's just a movement. The benefit of these things, I've already listed on the blog article.

+ Increase a little frame rate (really small compared to force decode on Image/IO), because we pre-transfer the bitmap to Render Process.
+ If the original CGImage's bitmap colorSpace does not match the iOS device screen native colorSpace. (For example, iPhone 6 use sRGB, iPhone 7 use P3), this can do a transform to match to avoid Core Animation do the samething during Render.
+ (?) Reduce the visual number of Xcode App memory.

Reply:
Emm, this is the common sense. You don't get what I mean. Please  check `sd_rawWebpImageWithData`  again.

Reply:
We just make the decompression in advance. And we may decompress twice, this is what I want to express.

Reply:
What is taht twice ? Actually. If you can , contact me in Slack or WeChat.

Reply:
Just, don't reply something unrelated to this PR. I just want you to update the comments...The code here is OK. So that I can release right now. Or I have to add these comments by myself again...